pennylane/measurements/vn_entropy.py

```diff
@@ -61,7 +61,7 @@ def circuit_entropy(x):
 
     .. note::
 
-        Calculating the derivative of :func:`~.vn_entropy` is currently supported when
+        Calculating the derivative of :func:`~pennylane.vn_entropy` is currently supported when
         using the classical backpropagation differentiation method (``diff_method="backprop"``)
         with a compatible device and finite differences (``diff_method="finite-diff"``).
 
```

Comment:
Should we still be pointing to `pennylane.qinfo.transforms.vn_entropy` ?

Reply:
It was always pointing to `qml.vn_entropy`, but I had to change this because sphinx was not able to reconcile which target we were referring to.

Reply:
Sorry, wasn't able to comment on line 68.  I was speaking about the line just below.

Reply:
Good idea, I'll get rid of it here and in other MPs.